lms/templates/calculator/toggle_calculator.html

```diff
@@ -9,15 +9,16 @@
   <div id="calculator_wrapper">
     <form id="calculator">
       <div class="input-wrapper">
-        <input type="text" id="calculator_input" title="${_('Calculator Input Field')}" tabindex="-1" />
+        <label for="calculator_input" class="sr">${_('Calculator Input Field')}</label>
+        <input type="text" id="calculator_input" title="${_('Calculator Input Field')}" />
 
         <div class="help-wrapper">
           <p class="sr" id="hint-instructions">${_('Use the arrow keys to navigate the tips or use the tab key to return to the calculator')}</p>
 
-          <a id="calculator_hint" href="#" role="button" aria-haspopup="true" tabindex="-1" aria-describedby="hint-instructions">${_("Hints")}</a>
+          <button id="calculator_hint" aria-haspopup="true" aria-controls=“calculator_input_help” aria-describedby="hint-instructions"><span class="sr">${_("Hints")}</span></button>
 
           <ul id="calculator_input_help" class="help" aria-activedescendant="hint-moreinfo" role="tooltip" aria-hidden="true">
```

Comment:
@ahsan-ul-haq see the first part of https://openedx.atlassian.net/browse/AC-107.  We need to get rid of role=tooltip and aria-activedescendent here.